web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,14 +1,20 @@
-.paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
-    }
+.field--type-image {
+  width: 50%;
+
+  img {
+    width: 100%;
+    height: 100%;
   }
+}
 
+.inline-field-label {
   .field__label {
     font-weight: bold;
-    color: $primary;
     margin-left: -$unit;
+    font-size: $small-font;
+  }
+
+  .field__value {
+    font-size: $small-font;
   }
 }
```

Comment:
Doesn't feel like these belong here, look like they should be in a field or label template or something like that definitely not here